xgboost_ray/tests/test_matrix.py

```diff
@@ -110,6 +110,39 @@ def testFromModinDfString(self):
         in_df["label"] = self.y
         self._testMatrixCreation(in_df, "label")
 
+    def testFromPetastormParquetString(self):
```

Comment:
hmm since petastorm is automatically enabled, would some previous tests be implicitly changed to leverage petastorm when we should instead leave it to test the non-petastorm codepath?

I didn't actually check the tests - but just a thought :)

Reply:
They won't, as petastorm requires an explicit `file://` prefix - and we don't have that in the other tests :-)